config/ola.m4

```diff
@@ -71,17 +71,17 @@ elif test -n "$1" ; then
   fi
 fi
 
-AC_ARG_WITH([ola-protoc],
-  [AS_HELP_STRING([--with-ola-protoc=COMMAND],
-    [use the given ola_protoc command instead of building one (useful for cross-compiling)])],
-  [],[with_ola_protoc=no])
+AC_ARG_WITH([ola-protoc-plugin],
+  [AS_HELP_STRING([--with-ola-protoc-plugin=COMMAND],
+    [use the given ola_protoc_plugin instead of building one (useful for cross-compiling)])],
+  [],[with_ola_protoc_plugin=no])
 
-OLA_PROTOC="\$(PROTOC) --plugin=protoc-gen-cppservice=\$(top_builddir)/protoc/ola_protoc_plugin${EXEEXT}";
 
-if test "$with_ola_protoc" != "no"; then
-  OLA_PROTOC=$with_ola_protoc;
-  echo "set ola_protoc to $with_ola_protoc"
+if test "$with_ola_protoc_plugin" != "no"; then
+  OLA_PROTOC="\$(PROTOC) --plugin=protoc-gen-cppservice=${with_ola_protoc_plugin}${EXEEXT}";
```

Comment:
You probably don't want the EXEEXT on the end (although you could work around it), looking at how it worked before and I guess what used to be passed in.